controllers/grid/files/SelectableSubmissionFileListCategoryGridHandler.inc.php

```diff
@@ -42,7 +42,7 @@ class SelectableSubmissionFileListCategoryGridHandler extends CategoryGridHandle
 	 * @param $capabilities integer A bit map with zero or more
 	 *  FILE_GRID_* capabilities set.
 	 */
-	function SelectableSubmissionFileListCategoryGridHandler($dataProvider, $stageId, $capabilities) {
+	function SelectableSubmissionFileListCategoryGridHandler($dataProvider, $stageId, $capabilities = 0) {
```

Comment:
No, it's not necessary, but it makes sense to have a 0 default in any bitfield IMO.